Formula/z/zig.rb

```diff
@@ -24,7 +24,6 @@ class Zig < Formula
   depends_on "cmake" => :build
   depends_on "lld@20"
   depends_on "llvm@20"
-  depends_on macos: :big_sur # https://github.com/ziglang/zig/issues/13313
```

Comment:
Why are we removing this?

Reply:
I removed it because Big Sur is dated by now

Reply:
Oh I read that wrong, I'll add back